app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
```

Comment:
@geekyd it overflows between screen size (780-769) also i have added compact class please check https://pr-122-open-event-frontend.herokuapp.com/events/963d3ba5/tickets/discount-codes